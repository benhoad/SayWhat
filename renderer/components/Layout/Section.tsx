interface ISectionProps {
  className?: string;
  children?: any;
}

export default function Section({ children }: ISectionProps) {
  return (
    <>
      <div className="Section">{children}</div>

      <style jsx>{`
        .Section {
          height: 100%;
          overflow: hidden;
          overflow-y: scroll;
        }
      `}</style>
    </>
  );
}
