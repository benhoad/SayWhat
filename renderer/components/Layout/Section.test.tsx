import { render } from "@testing-library/react";

import Section from "./Section";

describe("Section", () => {
  it("renders", () => {
    expect.hasAssertions();

    const { container } = render(<Section>SectionContent</Section>);
    expect(container.firstElementChild).toMatchSnapshot();
  });
});
