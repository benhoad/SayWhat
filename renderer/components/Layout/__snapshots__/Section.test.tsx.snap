// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section renders 1`] = `
<div
  class="jsx-1967107867 Section"
>
  SectionContent
</div>
`;
